src/views/CustomTileDialog/ImportExport/enhancedImportExport.ts

```
@@ -34,40 +48,159 @@ export interface ImportResult {
+        sortedGroups[key] = groups[key];
+      });
+
+    // Sort custom tiles by group name and intensity for better organization
+    const sortedCustomTiles: CleanExportData['customTiles'] = {};
+    Object.keys(customTiles)
+      .sort()
+      .forEach((groupName) => {
+        sortedCustomTiles[groupName] = {};
+        // Sort intensity levels numerically
+        Object.keys(customTiles[groupName])
+          .sort((a, b) => parseInt(a) - parseInt(b))
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)